Kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -58,6 +58,10 @@ data:
         # - prometheus.io/port: If port is not 9102 use this annotation
         monitor_kubernetes_pods = false
 
+        ## Restricts Kubernetes monitoring to namespaces for pods. This will take effect when monitor_kubernetes_pods is set to true
+        ##   ex: monitor_kubernetes_pods_namespaces = ["default1", "default2", "default3"]
```

Comment:
Can we please make this more specific that this namespace is only for pod annotations meaning for pod annotations for pods running in those namespaces only?